components/reddit/sources/new-comments-on-a-subreddit/new-comments-on-a-subreddit.js

```diff
@@ -0,0 +1,89 @@
+const common = require("../common");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-comments-on-a-subreddit",
+  name: "New comments on a subreddit",
+  description:
+    "Emits an event each time a new comment is added to a subreddit.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    subreddit: {
+      propDefinition: [common.props.reddit, "subreddit"],
+    },
+    subredditPost: {
+      type: "string",
+      label: "ID36 of subreddit",
+      description:
+        "Pipedream will emit a new event when a comment is posted to the subreddit pointed by this ID36.",
+      optional: false,
+    },
+    numberOfParents: {
```

Comment:
I'd make this optional with the current default of 0 parent comments

Reply:
I set the default of 0 here, in the other two instances where the min value is 2, the default was set to 2.